bin/inx-chronicle/src/api/stardust/history/responses.rs

```diff
@@ -10,18 +10,16 @@ use crate::api::impl_success_response;
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct TransactionHistoryResponse {
     pub address: String,
-    pub transactions: Vec<Transfer>,
-    pub paging_state: Option<String>,
+    pub items: Vec<Transfer>,
```

Comment:
Since the elements of the vec are `Transfer`s, why not call this field `transfers` then? :trollface:

Reply:
Because @Alexandcoats came up with `Transfer`, but we could not agree on it yet. 😁